pkgs/applications/audio/mopidy/argos.nix

```diff
@@ -0,0 +1,68 @@
+{
+  stdenv,
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  pkg-config,
+  meson,
+  ninja,
+  appstream-glib,
+  desktop-file-utils,
+  python3,
+  wrapGAppsHook3,
+  gobject-introspection,
+}:
+stdenv.mkDerivation rec {
+  pname = "mopidy-argos";
+  version = "1.14.0";
+
+  src = fetchFromGitHub {
+    owner = "orontee";
+    repo = "argos";
+    rev = "refs/tags/v${version}";
+    sha256 = "1G4o5gltRpgn4hu8+xBhx8YMjUwbmFRevFfmweQMFLA=";
+  };
+  pyproject = true;
```

Comment:
```suggestion
```
This doesn't actually do anything in a standard mkDerivation, I think. Didn't notice that earlier.

Reply:
It seems to build fine without it :+1: